package.json

```diff
@@ -142,7 +143,7 @@
     "preversion": "yarn clean",
     "test": "yarn build-client && PERCY_TOKEN=abc mocha --forbid-only \"test/**/*.test.ts\" --exclude \"test/percy-agent-client/**/*.test.ts\" --exclude \"test/integration/**/*\"",
     "test-client": "karma start ./test/percy-agent-client/karma.conf.js",
-    "test-integration": "yarn build-client && node ./bin/run exec -h localtest.me -- mocha test/integration/**/*.test.ts",
+    "test-integration": "yarn build-client && node ./bin/run exec -h *.localtest.me -- mocha test/integration/**/*.test.ts",
```

Comment:
Would this work? It's going to match all subdomains with that host name, but maybe not the hostname itself?

- `images-cdn.localtest.me` would match
- `css-cdn.localtest.me` would match
- `localtest.me` wouldn't match? 🤔

Reply:
According to [this](https://www.npmjs.com/package/domain-match), it should match. But I'll verify once I get some time to do so.

Reply:
https://github.com/percy/percy-agent/pull/347#discussion_r325840590